src/main/java/roomescape/reservation/controller/ReservationApiController.java

```diff
@@ -28,38 +30,62 @@ public ReservationApiController(ReservationService reservationService) {
     }
 
     @GetMapping("/reservations")
-    public ResponseEntity<List<ReservationResponse>> findAll() {
-        List<ReservationResponse> reservationResponses = reservationService.findAll();
+    public ResponseEntity<MultipleResponses<ReservationResponse>> findAll(@RequestParam(defaultValue = "SUCCESS") String status) {
```

Comment:
```suggestion
    public ResponseEntity<MultipleResponses<ReservationResponse>> findAll(@RequestParam(defaultValue = "SUCCESS") Status status) {
```

Enum으로 바로 받을 수도 있어요.

Reply:
오 .. Enum 타입으로 매핑도 되군요 👍🏻
해당 부분 반영했습니다 !